### DISCUSSION TITLE
single $'s for inline latex, i.e. $a=b$

### DISCUSSION THREAD

**Discussion by tkuraku** [damionx7/Zettel-Notes-Documentation#123] [Feature Requests]
GitHub Markdown uses single $ for inline latex, https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions, which is incompatible with zettel notes. I have a git repo with my knowledge base in markdown files. I view them with vscode on PC and zettel notes on mobile. It would be great if this incompatibility could be resolved.

Great product! I've purchased almost all the addons! Thank you!

**Comment by damionx7** [comment]
Thank you for your support ❤️. I have added this feature. It will be live in the next update.
